### PR TITLE
Add StandardSSD_LSR OS type and test case.

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

**Comment by squirrelsc** [comment]
@somil55 Please have a look on this change.

File: lisa/features/__init__.py [hunk 1]
```diff
@@ -1,7 +1,13 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
-from .disks import DiskEphemeral, DiskPremiumLRS, DiskStandardLRS, DiskType
+from .disks import (
+    DiskEphemeral,
+    DiskPremiumLRS,
+    DiskStandardLRS,
```
Comment (dsrivastavv) [inline lisa/features/__init__.py]: Maybe we should rename DiskStandardLRS to DiskStandardHDDLRS to differentiate scenario better. @LiliDeng @squirrelsc Thoughts?

Comment (LiliDeng) [reply lisa/features/__init__.py]: yep, I will make the change.

**Review by dsrivastavv — APPROVED** [review]

**Comment by LiliDeng** [comment]
@squirrelsc Could you please take a look?

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**